near/nearcore
# fix: memory leak in wasmer 2

Pulls in https://github.com/near/wasmer/pull/69

Test Plan
---------

* check that wasmer own tests unrelated to reference types/module linking pass
* check that we don't use refernce/types module linking https://github.com/near/nearcore/issues/5105
* fuzz test wasmer with patch vs wasmer without the patch overnight
* confirm using a soak test that the leak goes away
